.github/workflows/OCV-PR-docs.yaml

```diff
@@ -0,0 +1,105 @@
+name: OCV PR:docs
```

Comment:
@alalek, I see that this pipeline is common for all branches. Can we re-use it for all of them?

Reply:
It makes sense to bump image for 5.x branch based on Ubuntu 22.04.
Also regular nightly builds should do that for different deployment locations.

Also builds reporting on GitHub doesn't care about branches - we may have all-in-one mess. We could keep it if it doesn't bother.

Reply:
Because we should use modern software.
Old software may be used only if there are compatibility issues with existed releases on other branches